### ISSUE TITLE
Counter looses function on restart.

### ISSUE THREAD

**Issue by IamStig** [McJtyMods/RFToolsUtility#47]
MC 1.16.3
Forge:1.16.3-34.1.25
rftoolsutility-1.16-3.0.11

I have a sequencer leading into a counter. The sequencer has 4 ticks enabled and is set to loop2 and restarts on pulse from a mana detector, it points to a counter set to 2 & current 15.
It is used to place TNT for a botania setup, the counter gives it time to place and detonate 1 TNT before a 2nd is triggered. This works fine until I log off.
The counter becomes inactive and inaccessible. I need to break and replace, I can then access the UI to set it to 2 (and output 15) and it runs fine again.
ADDITIONAL - decided to test leaving the area, went to The End and returned. The counters UI is accessible BUT it loses the Current strength value....it drops to 1, but the Counter remains at 2.
This is a little strange. Further Update - the actual current output doe not seem to change when the value changed ....the redstone reads as 15

**Comment by McJty** [comment]
Weird. I cannot reproduce this. Can you reproduce this problems in a simple test world with only rftools and related?

**Comment by IamStig** [comment]
OK so I did 2 things one was to create a new MC 1.16.3 & Forge 34.1.25 installation.
Added McJtylib 1.16-5.0.12, RFTools Base 1.16-2.0.6 & RFtools Utility 1.16-3.0.12 and nothing else.
I then copied my world over and checked the counter, I could not access it without breaking and replacing, it was then functioning. I teleported to the end matter receiver and returned to find the counter was not accessible without breaking and replacing. Also just to be sure I saved and exited, after breaking and replacing the counter, and reloaded...same result, could not access counter without breaking and replacing.
I then created a new creative world and placed the sequencer and counter and some redstone and set them to operate as before. I saved and exited and then reloaded. The counter was not accessible without breaking and replacing. (I tried adding a picture of the test setup, hope it worked)
![2020-11-03_22 43 17](https://user-images.githubusercontent.com/20489424/97981244-141d7f00-1e26-11eb-8004-23bb1d944b6f.png)
